content/applications/hr/fleet/new_vehicle.rst

```diff
@@ -36,84 +49,248 @@ This section of the vehicle form relates to the person who is currently driving
+  not visible, that is because :guilabel:`Company` was selected instead of :guilabel:`Individual`.
+- :guilabel:`Name`: enter the name of the driver or company in this field.
+- :guilabel:`Company Name`: using the drop-down menu, select the company the driver is associated
+  with.
+
+  If :guilabel:`Company` is selected for the :guilabel:`Individual` or :guilabel:`Company` field,
+  this field does not appear.
+- :guilabel:`Contact`: enter the contact information in this section.
+
+  If desired, the :guilabel:`Contact` field can be changed to a different type of contact. Click on
+  :guilabel:`Contact` to reveal a drop-down menu. The available options to select are
+  :guilabel:`Contact`, :guilabel:`Invoice Address`, :guilabel:`Delivery Address`,
+  :guilabel:`Follow-up Address`, or :guilabel:`Other Address`.
+
+  If desired, select one of these other options for the :guilabel:`Contact` field, and enter the
+  corresponding information.
+
+  If :guilabel:`Company` is selected for the  :guilabel:`Individual` or :guilabel:`Company` field,
+  this field is labeled :guilabel:`Address` and cannot be modified.
+
+- :guilabel:`Tax ID`: enter the driver or company's tax ID in this field.
+- :guilabel:`Job Position`: enter the driver's job position in this field. If :guilabel:`Company` is
+  selected for the :guilabel:`Individual` or :guilabel:`Company` field, this field does not appear.
+- :guilabel:`Phone`: enter the driver or company's phone number in this field.
+- :guilabel:`Mobile`: enter the driver or company's mobile number in this field.
+- :guilabel:`Email`: enter the driver or company's email address in this field.
+- :guilabel:`Website`: enter the driver or company's website address in this field.
+- :guilabel:`Title`: using the drop-down menu, select the driver's title in this field. The default
+  options are :guilabel:`Doctor`, :guilabel:`Madam`, :guilabel:`Miss`, :guilabel:`Mister`, and
+  :guilabel:`Professor`.
+
+  If :guilabel:`Company` is selected for the :guilabel:`Individual` or :guilabel:`Company` field,
+  this field does not appear.
+- :guilabel:`Tags`: using the drop-down menu, select any tags that apply to the driver or company.
+
+  To add a new tag, type in the tag, then click :guilabel:`Create "tag"`.
+
+  There is no limit to the number of tags that can be selected.
+
+.. image:: new_vehicle/create-driver.png
+   :align: center
+   :alt: The top portion of the create driver form.
+
+Contacts & addresses tab
+************************
+
+After completing the top-half of the :guilabel:`Create Driver` or :guilabel:`Create Future Driver`
+form, add any other contacts and addresses associated with the driver or company in this tab.
+
+To add a new contact, click the :guilabel:`Add` button, and a :guilabel:`Create Contact` pop-up
+window appears.
+
+Enter the following information on the form:
+
+- :guilabel:`Contact Type`: select the type of contact being added. Click the radio button next to
```

Comment:
i understand your reasoning here, but i'm not sure if this should have GUI, as it doesn't directly appear this way on the UI. Perhaps there's a different way to mention it, so as to keep in line with the established RST guidelines.

Reply:
This was a hard one! I know in other docs, it was suggested to have a gui in bullet lists even for an entry like this, so that it looks nice when reading it- even though it is a technical stretch for a gui label.  I removed it- it doesn't look too bad.